views/movies/edit-movie.hbs

```diff
@@ -0,0 +1,22 @@
+<h2>Edit the Movie</h2>
+
+<form action="/movies/{{movie._id}}" method="post">
```

Comment:
Only thing you missed here was the /edit.

``<form action="/movies/{{movie._id}}/edit" method="post">``